communication/slack-config/usergroups.yaml

```diff
@@ -43,14 +43,14 @@ usergroups:
       - Jason
 
   # Should match the membership of the following teams at all times:
-  # - https://git.k8s.io/security/#product-security-committee-psc
+  # - https://git.k8s.io/security/#security-response-committee-src
```

Comment:
matches with the changes from this PR: https://github.com/kubernetes/security/pull/128